src/main/steps/applicant1/number-of-applicants/content.ts

```diff
@@ -0,0 +1,63 @@
+import { TranslationFn } from '../../../app/controller/GetController';
+import { FormContent } from '../../../app/form/Form';
+import { isFieldFilledIn } from '../../../app/form/validation';
+
+const en = ({ required }) => ({
+  section: 'Applicant details',
+  title: 'Are you applying on your own, or with someone else?',
+  one: "I'm applying on my own",
+  two: "I'm applying with my spouse or civil partner",
+  three: "I'm applying with someone who is not my spouse or civil partner",
+  errors: {
+    noOfApplicants: {
+      required,
+    },
+  },
+});
+
+const cy = ({ required }) => ({
+  section: 'Applicant details (but in welsh)',
```

Comment:
let's be consistent with (in welsh) so that we can easily find and replace
```suggestion
  section: 'Applicant details (in welsh)',
```